<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Harshu" failures="1" tests="8" name="C:/Users/patar/IdeaProjects/Maven_Harshada_Patare" time="0.107" errors="0" timestamp="2025-04-09T19:20:20 IST">
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseH" time="0.006"/>
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseA" time="0.0"/>
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseM" time="0.014">
    <skipped/>
  </testcase> <!-- TestCaseM -->
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseM" time="0.0">
    <skipped/>
  </testcase> <!-- TestCaseM -->
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseM" time="0.0">
    <skipped/>
  </testcase> <!-- TestCaseM -->
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseM" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testng.bacicoftestng.PriorityExample.TestCaseM(PriorityExample.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </failure>
  </testcase> <!-- TestCaseM -->
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseZ" time="0.0"/>
  <testcase classname="testng.bacicoftestng.PriorityExample" name="TestCaseX" time="0.0"/>
</testsuite> <!-- C:/Users/patar/IdeaProjects/Maven_Harshada_Patare -->
